pages/page.py

```diff
@@ -32,16 +44,11 @@ def url_current_page(self):
 
     @property
     def page_title(self):
-        WebDriverWait(self.selenium, self.timeout).until(lambda s: s.title)
-        return self.selenium.title
+        return Wait(self.selenium, self.timeout).until(lambda s: s.title)
 
     def refresh(self):
         self.selenium.refresh()
 
-    def open(self, url_fragment):
-        self.selenium.get(self.base_url + url_fragment)
-        self.selenium.maximize_window()
```

Comment:
The `open` method was moved. The maximize window command is replaced with a fixed window size command in `conftest.py`.